src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -70,6 +71,11 @@ HydrofractureSolver< POROMECHANICS_SOLVER >::HydrofractureSolver( const string &
     setApplyDefaultValue( 0 ).
     setInputFlag( InputFlags::OPTIONAL );
 
+  Base::template addLogLevel< logInfo::CrossflowWarning >();
+  Base::template addLogLevel< logInfo::HydraulicAperture >();
+  Base::template addLogLevel< logInfo::SolverTimeStep >();
+  Base::template addLogLevel< logInfo::SolverNextDt >();
```

Comment:
SolverTimeStep and SolverNextDt should be already covered in SolverBase, no?

Reply:
I removed SolverTimeStep and I left tSolverNextDt because it's a different logLovel

Reply:
why is it different? I am not sure that's intended.

Reply:
should be addressed now